test/utils/cloud_provider_aws.go

```diff
@@ -177,16 +177,19 @@ func (p *awsVPC) Reapply(timeout time.Duration) error {
 	}
 	p.output = output
 	// Wait for servers on VMs come to live.
-	err = wait.Poll(time.Second*5, time.Second*240, func() (bool, error) {
-		for _, ip := range p.GetVMIPs() {
+	for _, ip := range p.GetVMIPs() {
+		err = wait.Poll(time.Second*10, time.Second*600, func() (bool, error) {
```

Comment:
Can we check for agent connection, rather than curl check?

Reply:
As discussed, will do that as part of another PR.